tools/quickstart/quickstart/cli.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+"""import private key or keystore file as parity account"""
+
+import sys
+import os
+import re
+import json
+from eth_account import Account
+import click
+
+
+class TrustlinesFiles:
+    def __init__(self, password_file, address_file, keystore_file):
+        self.password_file = password_file
+        self.address_file = address_file
+        self.keystore_file = keystore_file
+
+    def store(self, account, password):
+        # Instead of just copying the file, we encrypt it again since the
+        # file itself may use scrypt as key derivative function which
+        # parity can't handle
+        json_account = account.encrypt(password, kdf="pbkdf2")
+
+        os.makedirs(os.path.dirname(os.path.abspath(self.keystore_file)), exist_ok=True)
+        with open(self.keystore_file, "w") as f:
+            f.write(json.dumps(json_account))
+
+        with open(self.password_file, "w") as f:
+            f.write(password)
+
+        with open(self.address_file, "w") as f:
+            f.write(account.address)
+
+
+def import_keystore_file(trustline_files, keystore_input_file):
+    keyfile_dict = json.loads(open(keystore_input_file, "rb").read())
+
+    while 1:
+        password = click.prompt(text="Password", hide_input=True)
+        try:
+            account = Account.from_key(Account.decrypt(keyfile_dict, password))
+            break
+        except ValueError:
+            click.echo("The password you entered is wrong. Please try again.")
+
+    trustline_files.store(account, password)
+    sys.exit(0)
+
+
+def read_private_key():
+    while 1:
+        privkey = click.prompt(text="Private key (hex encodded)", hide_input=True)
```

Comment:
Maybe not hide input for people like Bernd who don't copy paste their private key and will struggle to enter it (idk what is best practice)

Reply:
We could also let him enter it twice :)

I thought about not hiding it, but then I decided to keep it hidden, because I assumed everyone would copy and paste it anyway. If we don't hide it, people will copy and paste it to some github issue if they run into problems. This is something I'd like to avoid.